dev-learning/appsettings.json

```diff
@@ -6,5 +6,9 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+
+  "ConnectionStrings": {
+    "dbConfig": "server=localhost;port=3306;database=devlearning;user=root;password=aaaa"
```

Comment:
So are you using mysql connection string from app config or app settings? Having a connection string in two places is bad, I guess it's fine to have it here but remove the one from `app.config`. Also, I would suggest having a connection string password value as `a` or empty because I think it's default set to one of these values.